compiler/src/codegen/transl_anf.re

```diff
@@ -260,8 +260,8 @@ module RegisterAllocation = {
       | MSet(b, i) =>
         MSet(apply_allocation_to_bind(b), apply_allocations(ty, allocs, i))
       | MAllocate(x) => MAllocate(x)
-      | MDrop(i) => MDrop(apply_allocations(ty, allocs, i))
-      | MIncRef(i) => MDrop(apply_allocations(ty, allocs, i))
+      | MDrop(i, a) => MDrop(apply_allocations(ty, allocs, i), a)
+      | MIncRef(i) => MIncRef(apply_allocations(ty, allocs, i))
```

Comment:
Was this a typo bug previously?

Reply:
Yes 🤦 

But all of this code is completely unused anyway.